src/vectorchord/getting-started/vectorchord-suite.md

```diff
@@ -0,0 +1,132 @@
+# VectorChord Suite
+
+Retrieval-Augmented Generation (RAG) is revolutionizing our interaction with vast datasets and language models. RAG systems offer more precise, contextually aware, and current answers by retrieving pertinent information before generating responses. Nonetheless, constructing robust, scalable, and efficient RAG pipelines poses considerable challenges, particularly for production use.
+
+Many solutions feature complex architectures, using distinct databases for vector search, keyword search, and primary data storage. This commonly results in data synchronization challenges, greater infrastructure complexity, and higher operational costs.
+
+What if you could build a powerful, production-ready RAG system directly within your trusted PostgreSQL database?
+
+Enter the **VectorChord Suite**, a collection of PostgreSQL extensions designed to bring high-performance vector search, native BM25 ranking, and flexible tokenization capabilities right into Postgres. Let's explore the components and how they enable next-generation RAG solutions.
+
+## What is the VectorChord Suite?
+
+The suite comprises three key PostgreSQL extensions working in concert:
+
+1. [**VectorChord:**](https://github.com/tensorchord/VectorChord) The core vector search engine. It's specifically designed for scalable, high-performance, and disk-efficient vector similarity search within PostgreSQL.
+    
+2. [**VectorChord-bm25:**](https://github.com/tensorchord/VectorChord-bm25) This extension implements the sophisticated BM25 ranking algorithm directly inside PostgreSQL, leveraging efficient Block-WeakAnd algorithms. BM25 is a standard for relevance ranking based on keyword frequency and document characteristics.
+    
+3. [**pg_tokenizer.rs:**](https://github.com/tensorchord/pg_tokenizer.rs) Provides essential text tokenization capabilities needed for effective full-text search, enabling fine-grained control over how text is processed for full-text search.
+    
+
+By combining these extensions, you unlock powerful capabilities for building advanced RAG systems entirely within PostgreSQL.
+
+## How to Use the VectorChord Suite
+
+You can use the `tensorchord/vchord-suite` Docker image to run multiple extensions which are provided by TensorChord. The image is based on the Docker official PostgreSQL image and includes the following extensions:
+- `vchord`
+- `pg_tokenizer` 
+- `vchord_bm25`
+
+```powershell
```

Comment:
Is docker command supported by powershell?

Reply:
change to shell